crates/supervisor/service/src/service.rs

```diff
@@ -0,0 +1,75 @@
+//! Contains the main Supervisor service runner.
+
+use anyhow::Result;
+use jsonrpsee::server::{ServerBuilder, ServerHandle};
+use kona_supervisor_core::{Supervisor, SupervisorRpc};
+use kona_supervisor_rpc::SupervisorApiServer;
+use std::{net::SocketAddr, sync::Arc};
+use tracing::{info, warn};
+
+/// Configuration for the Supervisor service.
+#[derive(Debug, Clone)]
+pub struct Config {
+    /// The socket address for the RPC server to listen on.
+    // TODO:: refactoring required. RPC config should be managed in it's domain
+    pub rpc_addr: SocketAddr,
+    // Add other configuration fields as needed (e.g., connection details for L1/L2 nodes)
+}
+
+/// The main service structure for the Kona Supervisor.
+/// orchestrates the various components of the supervisor
+#[derive(Debug)]
+pub struct Service {
+    config: Config,
+    _supervisor: Arc<dyn kona_supervisor_core::SupervisorService + Send + Sync>,
```

Comment:
```suggestion
    _supervisor: Arc<dyn SupervisorService + Send + Sync>,
```
no reason to use long path here